### ISSUE TITLE
 This plugin is not responding

### ISSUE THREAD

**Issue by repercussion** [plexinc-plugins/PBSKids.bundle#11]
I do not have this issue with other plugins, including the regular PBS plugin, but can't get any of the content to load.

![plugin not responding](https://i.imgur.com/0s6OYg5.png)

**Comment by mddepew** [comment]
Yeah, PBS Kids changed their api. The current plugin is broken. I have a pull request in with a fix.
https://github.com/plexinc-plugins/PBSKids.bundle/pull/10

If you want, you can get my fixed version here: 
https://github.com/mddepew/PBSKids.bundle

**Comment by sander1** [comment]
@mddepew Thanks for the update! I'm back at home tomorrow and will have a look at the update/pull in the update! Thanks!

**Comment by mddepew** [comment]
@Sander1 There are a few things, thumbnail grabbing specifically, that are done sub-optimally in my PR. Did you find a reference somewhere for the pbs kids api, or was it reverse engineered? I figured out what I could, but an api reference would be great.

**Comment by sander1** [comment]
I've only done some minor updates on this plugin. I don't know how people figured out the source(s) to use, but I think it's all based on looking at the website and see what http connections are done to get data.

**Comment by pdramos1** [comment]
You may want to check out this site. https://docs.pbs.org/category/dd
Looks like there potentially could be API documentation that would be useful etc. 
It would likely also alert you to any api changes so you could get out ahead of breaking changes.
